packages/appstore/src/components/modals/real-wallets-upgrade/components/wallets-upgrade-step-one/wallets-upgrade-step-one.tsx

```diff
@@ -0,0 +1,91 @@
+import { Button, Icon, Modal, Text } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import { useStore } from '@deriv/stores';
+import './wallets-upgrade-step-one.scss';
+import WalletsImage from 'Assets/svgs/wallets';
+
+type TWalletsUpgradeStepOneFooter = {
+    handleClose: VoidFunction;
+    handleNext: VoidFunction;
+};
+
+const WalletsUpgradeStepOneContent = () => {
+    const { ui } = useStore();
+    const { is_mobile } = ui;
+
+    const bullets = [
+        localize('Better funds segregation'),
+        localize('Instant transfers between Wallets and trading accounts'),
+        localize('Multiple currency support'),
+    ];
+
+    return (
+        <div className='wallets-upgrade-step-one__content'>
+            <div className='wallets-upgrade-step-one__image-container'>
+                <WalletsImage image={`why_wallets_${is_mobile ? 'mobile' : 'desktop'}`} />
+            </div>
+            <Text
+                as='h1'
+                color='prominent'
+                weight='bold'
+                align='center'
+                size={is_mobile ? 'xsm' : 'l'}
+                className='wallets-upgrade-step-one__title'
+                line_height={is_mobile ? 'm' : 'xs'}
+            >
+                <Localize i18n_default_text='Why Wallets' />
+            </Text>
+            <Text
+                as='p'
+                color='prominent'
+                size={is_mobile ? 's' : 'm'}
+                align='center'
+                className='wallets-upgrade-step-one__description'
+                line_height='m'
+            >
+                <Localize i18n_default_text='Deposit, transfer, trade' />
+            </Text>
+            <div className='wallets-upgrade-step-one__bullet-list-container'>
+                {bullets.map(bullet => (
+                    <div key={bullet} className='wallets-upgrade-step-one__bullet'>
+                        {bullet && (
+                            <div className='wallets-upgrade-step-one__bullet-row'>
+                                <Icon
+                                    icon='IcAppstoreTick'
+                                    className='wallets-upgrade-step-one__bullet-icon'
+                                    size={is_mobile ? 12 : 16}
+                                />
+                                <Text
+                                    as='p'
+                                    color='prominent'
+                                    align='center'
+                                    className='wallets-upgrade-step-one__bullet-text'
+                                    size={is_mobile ? 'xs' : 's'}
+                                    line_height={is_mobile ? 's' : 'm'}
+                                >
+                                    {bullet}
+                                </Text>
+                            </div>
+                        )}
+                    </div>
+                ))}
```

Comment:
Can we seperate this and make it into a sub component or something?

Reply:
Done